discussions/tasks.py

```diff
@@ -57,6 +57,23 @@ def sync_discussion_users():
             log.error('Impossible to sync user_id %s to discussions', user_id)
 
 
+@app.task()
+def force_sync_discussion_users():
+    """
+    Sync the user's profile to open discussions along with email optin flag
+    """
+    if not settings.FEATURES.get('OPEN_DISCUSSIONS_USER_SYNC', False):
+        log.debug('OPEN_DISCUSSIONS_USER_SYNC is set to False (so disabled) in the settings')
+        return
+
+    users_to_backfill = Profile.objects.values_list('user__id', flat=True)
+    for user_id in users_to_backfill:
+        try:
+            api.create_or_update_discussion_user(user_id, allow_email_optin=True)
+        except DiscussionUserSyncException:
```

Comment:
Can you add a unit test for `force_sync_discussion_users ` where `api.create_or_update_discussion_user` raises a `DiscussionUserSyncException`, and verify that the log message is `"Impossible to sync user_id <id> to discussions"`? Similar to the test `test_sync_discussion_users_task_api_error`